cmd/fetchbranches/CI.go

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Semaphore environment variables https://semaphoreci.com/docs/available-environment-variables.html
+
+package main
+
+import "os"
+
+// CI Continous Integration
+type CI interface {
+	// GetPR will obtain the pull request number
```

Comment:
please replace all ```will + verb``` with ```verb```
here is an example https://golang.org/src/os/exec.go?s=2947:3026#L118
```
// Signal sends a signal to the Process.
```